resources/dicts/patrols/general/training.json

```diff
@@ -9673,5 +9673,264 @@
+                "exp": 0,
+                "weight": 10,
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["platonic", "comfort", "trust"],
+                        "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "r_c completes the training session, but stumbles out of the forest and collapses on a field of grass.",
+                "exp": 0,
+                "weight": 10,
+                "dead_cats": ["r_c"],
+                "history_text": {
+                       "reg_death": "m_c was killed after training in the Dark Forest."
+                }
+            }
+        ]
+    },    
+    {
+        "patrol_id": "gen_train_solodarkforest",
+        "biome": ["Any"],
+        "season": ["Any"],
+        "types": ["training"],
+        "tags": [],
+        "patrol_art": "gen_train_darkforest2",
+        "min_cats": 1,
+        "max_cats": 1,
+        "min_max_status": {
+            "all apprentices": [-1, -1]
+        },
+        "pl_skill_constraint": ["DARK,2"],
+        "weight": 20,
+        "intro_text": "p_l stands at the entrance of a gloomy forest, draped in mud. A spirit waits, fur puffed up, claws unsheathed.",
+        "decline_text": "p_l turns tail and races away! {PRONOUN/p_l/poss} failure is jeered and laughed at by all the other Dark Forest spirits.",
+        "chance_of_success": 65,
+        "success_outcomes": [
+            {
+                "text": "p_l is met with a spirit who growls at {PRONOUN/p_l/object}. p_l pounces on them and wrestles them, and the two scuffle. p_l sinks {PRONOUN/p_l/poss} teeth in the Dark Forest spirit's neck, and they hiss, their body melting beneath p_l. Their pelt fades until there is no more, and p_l is alone, but not unhurt.",
+                "exp": 15,
+                "weight": 20,
+                "injury": [
+                    {
+                        "cats": ["p_l"],
+                        "injuries": ["cat bite", "claw-wound"],
+                        "scars": ["BRIGHTHEART", "SIDE", "THREE", "MANTAIL", "RIGHTBLIND"]
+                    }
+                ],
+                "history_text": {
+                    "reg_death": "m_c was killed after training in the Dark Forest.",
+                    "lead_death": "after a brutal training session in the Dark Forest left them dead",
+                    "scar": "m_c was scarred after a brutal training session in the Dark Forest."
+                }
+            },
+            {
+                "text": "p_l scowls as {PRONOUN/p_l/subject} feel the scratches along {PRONOUN/p_l/poss} fur ache. {PRONOUN/p_l/subject/CAP} swipe claws at the spirit's throat, causing them to die and fade. p_l ambles to camp with a stumble in {PRONOUN/p_l/poss} step, collapsing at camp.",
```

Comment:
ambles is the wrong vibe to give here, it's a relaxed walk. Maybe p_l drags themself to camp with a stumble in their step?

Reply:
Also I'd not have the spirit die here, give hiss and run off or something